test/extended/prometheus/prometheus.go

```diff
@@ -438,7 +430,7 @@ var _ = g.Describe("[sig-instrumentation] Prometheus [apigroup:image.openshift.i
 					targets.Expect(labels{"job": "node-exporter"}, "up", "^https://.*/metrics$"),
 					targets.Expect(labels{"job": "prometheus-operator"}, "up", "^https://.*/metrics$"),
 					targets.Expect(labels{"job": "alertmanager-main"}, "up", "^https://.*/metrics$"),
-					targets.Expect(labels{"job": "crio"}, "up", "^http://.*/metrics$"),
+					targets.Expect(labels{"job": "crio"}, "up", "^http(s)?://.*/metrics$"),
```

Comment:
Do we need to make sure `http` works too? if we are only going migrate to `https`  then `(s)?` maybe redundant (and simply having `https` might be sufficient)?

Reply:
There is a chicken and egg problem. We have to merge this regex to support both, then do another PR after everything else merges to switch it to https only.

Reply:
Aha, ok. Thanks.

Reply:
BTW, unless you want to do something with the grouping, IMO you `https?` might also work.